_posts/2020-01-26-Pycon-de-writeup.markdown

```diff
@@ -0,0 +1,273 @@
+
+### Title: Automating feature engineering for supervised learning? Methods, open-source tools and prospects.
+
+Link: [https://www.youtube.com/watch?v=XK7mKGWSue8](https://www.youtube.com/watch?v=XK7mKGWSue8)
+
+Speaker: Thorben Jensen
+
+Summary: The talk basically reported a case study comparing three open source libraries ``tsfresh``, ``featuretools`` and ``TPOT``. At the end of the day, ``TPOT`` seemed superior in most categories, including the degree of automation. However, the case study was only carried out on a single data set. When asked about the potential biases of this dataset, the speaker said that ``tsfresh`` does provide the better support for time series problems, and that this may be more critical for other datasets.
+(Shameless plug: ``tsfreh`` is a very popular Blue Yonder open source project: [https://github.com/blue-yonder/tsfresh](https://github.com/blue-yonder/tsfresh))
+
+### Title: Docker and Python - A Match made in Heaven
+
+Link: [https://www.youtube.com/watch?v=Q2u1wcfmlzw](https://www.youtube.com/watch?v=Q2u1wcfmlzw)
+
+Speaker: Dr. Hendrik Niemeyer
+
+Summary: A very detailed and well structured introduction into docker and its usage for python development. Many pointers to useful tools and frameworks and some good introductory live demos.
+
+### Title: 10 Years of Automated Category Classification for Product Data
+
+Link: [https://www.youtube.com/watch?v=R-ts_mxGKOg](https://www.youtube.com/watch?v=R-ts_mxGKOg)
+
+Speaker: Johannes Knopp
+
+Summary: Johannes shows the historic evolution of their product category classification at "solute GmbH", a Karlsruhe based competitor prices service provider. I love how he openly talks about failed attempts, historic quirks and problems they faced over the years.
+
+### Title: Dr. Schmood's Notebook of Python Calisthenics and Orthodontia
+
+Link: [https://www.youtube.com/watch?v=BAsL_cbkQDc](https://www.youtube.com/watch?v=BAsL_cbkQDc)
+
+Speaker: David Schmudde
+
+Summary: I have to admit, the title would not have pulled me into this talk as I have no clue what the words are about. As I was session chair, I was lucky enough to attend it. In fact, it was about the state handling in python and why some complain about "messed up variable states" in IPython notebooks. He also suggests to embrace immutability to solve the situation and I love immutability!
+
+### Title: Python 2020+
+
+Link: [https://www.youtube.com/watch?v=fOdCxum-qLA](https://www.youtube.com/watch?v=fOdCxum-qLA)
+
+Speaker: Łukasz Langa
+
+Summary: In this keynote, Łukasz gives a unique look behind the scenes of how Python, especially cpython, is developed and he gives some predictions where the future will, or should go. But you should definitively watch the talk because of the many anecdotes shared by him. BTW: In the hallway track, Łukasz, the creator of "black" assured me he also doesn't like some of the formatting decisions, but it is "black". Even he is not able to argue with it, it's "black"!
```

Comment:
```suggestion
Summary: In this keynote, Łukasz gives a unique look behind the scenes of how Python, especially CPython, is developed and he gives some predictions where the future will, or should go. But you should definitively watch the talk because of the many anecdotes shared by him. BTW: In the hallway track, Łukasz, the creator of "black" assured me he also doesn't like some of the formatting decisions, but it is "black". Even he is not able to argue with it, it's "black"!
```